advanced_source/dynamic_quantization_tutorial.py

```diff
@@ -188,44 +185,42 @@ def tokenize(self, path):
     print(all_output)
 
 ######################################################################
-# It's no GPT-2, but it looks like the model has started to learn the structure of
-# language!
+# 이 모델이 GPT-2는 아니지만, 언어의 구조를 배우기 시작한 것처럼 보입니다!.
 #
-# We're almost ready to demonstrate dynamic quantization. We just need to define a few more
-# helper functions:
+# 동적 양자화를 설명하기 위한 준비가 거의 다 됐습니다. 몇 개의 보조 함수만 정의하면 됩니다.
 
 bptt = 25
 criterion = nn.CrossEntropyLoss()
 eval_batch_size = 1
 
-# create test data set
+# 시험용 데이터 셋을 생성합니다.
 def batchify(data, bsz):
-    # Work out how cleanly we can divide the dataset into bsz parts.
+    # 얼마나 깔끔하게 데이터셋을 bsz 개의 부분으로 나눌 수 있는지 계산합니다.
     nbatch = data.size(0) // bsz
-    # Trim off any extra elements that wouldn't cleanly fit (remainders).
+    # 데이터에서 쓰이지 않는 잉여 부분들을 제거합니다.
     data = data.narrow(0, 0, nbatch * bsz)
-    # Evenly divide the data across the bsz batches.
+    # 데이터를 bsz로 등분합니다.
     return data.view(bsz, -1).t().contiguous()
 
 test_data = batchify(corpus.test, eval_batch_size)
 
-# Evaluation functions
+# 평가 함수들
 def get_batch(source, i):
     seq_len = min(bptt, len(source) - 1 - i)
     data = source[i:i+seq_len]
     target = source[i+1:i+1+seq_len].view(-1)
     return data, target
 
 def repackage_hidden(h):
-  """Wraps hidden states in new Tensors, to detach them from their history."""
+  """은닉 상태를 기록으로부터 떼어내기 위해 새로운 tensor로 감쌉니다."""
```

Comment:
뗴어낸다기보단, 제거한다는 표현이 좋지 않을까요? 새로운 Tensor로 감싸는 표현보다는 재구성한다는 표현도 좋아보입니다 :)

Reply:
```suggestion
  """은닉 상태를 변화도 기록에서 제거된 새로운 tensor로 만듭니다."""
```
텐서의 구성이 바뀐다기보단 속성이 바뀌는 느낌이 들어서 함수 이름도 repackage이지만 단순하게 만든다... 라고 했습니다.